library/ssl_tls.c

```diff
@@ -8231,6 +8231,9 @@ int mbedtls_psa_ecjpake_read_round_one(
         }
     }
 
+    if ( input_offset != len )
+        return PSA_ERROR_INVALID_ARGUMENT;
```

Comment:
I think that should be `MBEDTLS_ERR_SSL_HANDSHAKE_FAILURE` (or another `MBEDTLS_ERR_SSL_` error code) not a `PSA_ERROR_` value, for consistency with other error paths in this function. (Same for the other parsing function.)